openedx/features/data_extract/helpers.py

```diff
@@ -49,17 +52,17 @@ def get_teams_data(course_key):
     course_teams = CourseTeam.objects.filter(course_id=course_key)
     team_data = []
     for team in course_teams:
-        team_data.push({
-            'team_id': course_team.team_id,
-            'name': course_team.name,
-            'course_id': course_team.course_id.to_deprecated_string(),
-            'topic_id': course_team.topic_id,
-            'date_created': course_team.date_created.__str__(),
-            'description': course_team.description,
-            'country': course_team.country,
-            'language': course_team.language,
-            'last_activity_at': course_team.last_activity_at.__str__(),
-            'team_size': course_team.team_size,
+        team_data.append({
```

Comment:
These are my changes that I made in osama/LP-845 few weeks ago. Why are the showing up here?